src/Resources/public/google-places-filter/google-places-filter.js

```diff
@@ -191,12 +191,12 @@ var GooglePlacesFilter = (function () {
         };
 
         var resetRadiusField = function (place) {
-            if (place && filter.dom.value !== '' && (place.types.includes('street_address') || place.types.includes('route') || place.types.includes('premise'))) {
+            if (place.types && filter.dom.value !== '' && (place.types.includes('street_address') || place.types.includes('route') || place.types.includes('premise'))) {
```

Comment:
If `place` is not existing, from my point of view also here an error is thrown. Could you please add a double check?
```js
if(place && place.types && ...)
```

Reply:
```javascript
// or you initialize the function variable
const foo = function (bar=false) {
    console.log(typeof bar);  // boolean
    console.log(typeof bar.sub); // undefined
}
foo();
```

Reply:
Could you please point out a concrete example? I am unsure what exactly you want to achieve with this.